playbooks/common/openshift-cluster/upgrades/library/openshift_upgrade_config.py

```diff
@@ -78,6 +78,10 @@ def upgrade_master_3_0_to_3_1(ansible_module, config_base, backup):
         config['kubernetesMasterConfig'].pop('apiLevels')
         changes.append('master-config.yaml: removed kubernetesMasterConfig.apiLevels')
 
+    # Add masterCA to serviceAccountConfig
+    if 'serviceAccountConfig' in config and 'masterCA' not in config['serviceAccountConfig']:
+        config['serviceAccountConfig']['masterCA'] = 'ca.crt'
```

Comment:
I know during installation we set the name of the certificate to `ca.crt` but for upgrades should this be assumed or should we default it to another value?  Should we check for it or would it be safer to take the value from `oauthConfig.masterCA` since we know it likely exists?